_posts/2016-05-20-tyxml4.md

```diff
@@ -0,0 +1,69 @@
+---
+title: TyXML 4.0.0
+layout: default
+author: The Ocsigen Team
+authorurl: https://ocsigen.org
+---
+
+This is with great pleasure that we are announcing the release of [TyXML 4.0.0][changelog]. This new version is focused on improving the user experience, both for old and new TyXML users.
+
+[changelog]: https://github.com/ocsigen/tyxml/releases/tag/4.0.0
+
+## What is TyXML ?
+
+TyXML is a library for building statically correct HTML5 and SVG documents.
+It provides a set of combinators which use the OCaml type system to ensure the validity of the generated document. TyXML's combinators can be used to build textual HTML and SVG, but also DOM trees or reactive interfaces, using [Eliom][] and [Js_of_ocaml][].
+
+[Eliom]: https://ocsigen.org/eliom/manual/clientserver-html
+[Js_of_ocaml]: https://ocsigen.org/js_of_ocaml/api/Tyxml_js
+
+## New TyXML manual and improved documentation
+
+A new TyXML manual is now available [here][manual]. Documentation was also improved in the various TyXML modules. Do not hesitate to provide feedback on [our bug tracker][bugtracker]!
```

Comment:
The documentation of the various TyXML modules was also improved.

Feedback *via* our bug tracker (sounds like you are asking feedback on the tracker itself).